libs/net/models/Areas/Subscriber/Models/Product/UserProductModel.cs

```diff
@@ -109,15 +110,13 @@ public UserProductModel(Entities.UserProduct entity) : base(entity)
         else if (entity.Product.ProductType == Entities.ProductType.Notification)
         {
             var subscription = entity.User.NotificationSubscriptionsManyToMany
-                .FirstOrDefault(s => s.UserId == entity.UserId &&
-                    s.NotificationId == entity.Product!.TargetProductId);
+                .FirstOrDefault(s => s.IsSubscribed && s.NotificationId == entity.Product!.TargetProductId);
             this.IsSubscribed = subscription?.IsSubscribed ?? false;
         }
         else if (entity.Product.ProductType == Entities.ProductType.EveningOverview)
         {
             var subscription = entity.User.AVOverviewSubscriptionsManyToMany
-                .FirstOrDefault(s => s.UserId == entity.UserId &&
-                    (int)s.TemplateType == entity.Product!.TargetProductId);
+                .FirstOrDefault(s => s.IsSubscribed && (int)s.TemplateType == entity.Product!.TargetProductId);
```

Comment:
I don't love this, but since a user can be subscribed through a distribution list it's currently necessary.